src/include/storage/index/index.h

```diff
@@ -0,0 +1,108 @@
+#pragma once
+
+#include <utility>
+#include <vector>
+#include "catalog/catalog_defs.h"
+#include "common/performance_counter.h"
+#include "storage/index/compact_ints_key.h"
+#include "storage/index/generic_key.h"
+#include "storage/index/index_defs.h"
+#include "storage/index/index_metadata.h"
+#include "storage/storage_defs.h"
+
+// clang-format off
+#define IndexCounterMembers(f) \
```

Comment:
We're not really using performance counters right now. Are we leaving them here as an example or should we get rid of them?

Reply:
Removed.